source/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication.md

```diff
@@ -82,17 +84,15 @@ We don't specifically recommend one, but here are a few of many SSL providers:
 - [TrustWave](https://ssl.trustwave.com/support/install-certificate-cpanel.php "TrustWave")
 - [VeriSign](http://www.verisigninc.com/ "VeriSign")
 
-
-
 ## Load Certificate into Pantheon
 
 1. From your Site Dashboard, select **Dev**, **Test**, or **Live** (most commonly **Live**).
-2. Select **Domains**.
-2. Select **SSL**.
-3. Paste in the requested information and press **Add Cert**.
- ![Site dashboard add SSL certificate step 2](/source/docs/assets/images/desk_images/259882.png)​
+2. Select **Domains / SSL**.
+2. Click **SSL**.
+3. Paste in the certificate information, including the header and footer, and press **Add Cert**.
```

Comment:
What do you mean header and footer? (I know what you mean, but I think it needs clarification)